packages/components/src/particles/Card/index.tsx

```diff
@@ -4,13 +4,15 @@ import { createMemo, For, JSX } from 'solid-js';
 export interface CardProps extends JSX.HTMLAttributes<HTMLDivElement> {
   children: JSX.Element | JSX.Element[];
   selected?: boolean;
+  gray?: boolean;
```

Comment:
I think I would do `variant?: "gray"` so that in the future its easy to add variants

Reply:
yup makes sense